### ISSUE TITLE
Just manual toggle

### ISSUE THREAD

**Issue by memeplex** [robbawebba/night-light#17]
Please add possibility to get just manual toggling *without* automatic reswitching afterwards. I move from one environment to another and light conditions change quite irrespectively of time and location.

Maybe this is already possible without horrible hacks like setting sunset/rise times that states: there is no sun here, never, ever. But I mean: a cleaner way.

**Comment by robbawebba** [comment]
@memeplex Thanks for your feedback! Just to clarify - you would like to disable all automatic theme switching so that you could manually toggle between themes?

**Comment by memeplex** [comment]
Exactly, now I can invert the last automatic switch until the next one, but not permanently.
